File: src/main/kotlin/net/namibsun/nuztrack/routes/runs/RunsTransfer.kt
package net.namibsun.nuztrack.routes.runs

data class NuzlockeRunTO(
        val id: Long,
        val userName: String,
        val name: String,
        val game: String,
        val rules: List<String>,
        val customRules: List<String>,
        val status: String
)

data class CreateNuzlockeRunTO(
        val name: String,
        val game: String,
        val rules: List<String>,
        val customRules: List<String>
)

File: frontend/src/api/runs/runsTransfer.ts
export type NuzlockeRunTO = {
    id: number,
    username: string,
    name: string,
    game: string,
    rules: string[],
    customRules: string[],
    status: string
}

export type CreateNuzlockeRunTO = {
    name: string,
    game: string,
    rules: string[],
    customRules: string[]
}
